sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -1610,6 +1610,26 @@ object SQLConf {
       .checkValue(v => Set(1, 2).contains(v), "Valid versions are 1 and 2")
       .createWithDefault(2)
 
+  val STREAMING_SESSION_WINDOW_MERGE_SESSIONS_IN_LOCAL_PARTITION =
+    buildConf("spark.sql.streaming.sessionWindow.merge.sessions.in.local.partition")
```

Comment:
How about `spark.sql.streaming.sessionWindow.localMerge.enabled` or `spark.sql.streaming.sessionWindow. mergeSessionsInLocalPartition.enabled`?

Reply:
Comparing with the similar logic of AggUtils, maybe we can also remove this config? Just always do the local merge?

Reply:
Yea, is it necessary to have this config? Seems we can always do it.

Reply:
As I explained on the doc method, this would incur additional "logical sort", so only useful when there're lots of input rows which are going to be consolidated into same session. The benefit is dependent on the characteristic of data.

If we want to pick one between two to simplify, it would be probably safer to remove local aggregation.